JS.md

```diff
@@ -29,6 +29,21 @@ Because JavaScript exceptions can be handled, and JavaScript can continue to
 call WebAssembly exports after a trap has been handled, traps do not, in
 general, prevent future execution.
 
+Even though the `stack` property of JS `Error` objects is not specified by
+ES-262, implementations *should* include WebAssembly stack frames in
+the generated callstacks. [User-defined sections](BinaryEncoding.md#high-level-structure)
+may arbitrarily affect the displayed stack frames. However, by default, when
+no relevant sections are present:
+* The line number reported for a WebAssembly frame *should* be the offset of the 
+  the trap/call instruction within the module's originally-compiled binary.
+* The column number reported for WebAssembly frames, if not omitted entirely,
+  *should* be 1.
+* The function name *should* be generated from the template string literal 
+  `wasm-function[${i}]` where `i` is the index of the function in the module's
+  [Code section](BinaryEncoding.md#code-section).
```

Comment:
The intention was just to be able to syntactically distinguish the wasm functions from the normal JS functions since you'll see them mixed into the same callstack.  I did indeed forget to explicitly mention the Names section here; I'll add that.